.github/workflows/ci.yml

```diff
@@ -130,3 +130,35 @@ jobs:
         name: Check artifacts
         run: |
           find ${{ env.DESTDIR }} -type f -exec file -e ascii -- {} +
+
+  govulncheck:
+    runs-on: ubuntu-24.04
```

Comment:
Reminds me; I see the other ones are still on 20.04, and some on "ubuntu-latest".

Reply:
Ah right we should pin to `ubuntu-24.04` where it's possible imo.

Reply:
Yeah, It's fine to do that separately and have this one already on 24.04 I guess.

I thought I had a PR to do that, but maybe I never opened one 🙈